components/Footer.tsx

```diff
@@ -12,28 +13,16 @@ export const Footer: React.FC = () => {
         <Navbar className="justify-content-between" variant="dark">
           <Nav className="flex-column flex-sm-row">
             <Nav.Link href="#top">{t('footer.toTop')}</Nav.Link>
-            <Nav.Link target="_blank" rel="noopener noreferrer" href="https://www.edd.ca.gov/About_EDD/Contact_EDD.htm">
+            <Nav.Link target="_blank" rel="noopener noreferrer" href={getUrl('edd-about-contact')}>
               {t('footer.contact')}
             </Nav.Link>
-            <Nav.Link
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://www.edd.ca.gov/About_EDD/Conditions_Of_Use.htm"
-            >
+            <Nav.Link target="_blank" rel="noopener noreferrer" href={getUrl('edd-about-conditions-of-use')}>
```

Comment:
I see that we have this pattern a lot:

```
<Nav.Link target="_blank" rel="noopener noreferrer" href={something}>
```

Is it worth breaking into its own mini component?

Reply:
maybe? if we use it going forward I'm game but I don't feel like it warrants a refactor to do so!